docs/advanced/settings.md

```diff
@@ -427,6 +427,14 @@ This setting lets you define the support bundle's default timeout duration in mi
 
 Default: `10`
 
+## `support-bundle-expiration`
+
+_Available as of v1.3.0_
+
+This setting lets you define the support bundle's default expiration duration in minutes. The minimum value is `30` minutes. It deletes unused support bundle if support bundle finished packaging, but no one downloads.
```

Comment:
```suggestion
This setting allows you to define the number of minutes Harvester waits before deleting a support bundle that has been packaged but not downloaded. The minimum value is `30`.
```

Reply:
> expiration duration

This is confusing because "expiration" implies a deadline while "duration" implies a range. I reworded it to reflect the actual system behavior.